orchestration/dags/data_gcp_dbt/models/analytics/enriched/enriched_offer_data.sql

```diff
@@ -154,6 +154,13 @@ SELECT
         ) THEN TRUE
         ELSE FALSE
     END AS offer_is_underage_selectable,
+    CASE
+            WHEN subcategories.category_id = 'MUSIQUE_LIVE' THEN "MUSIC"
+            WHEN subcategories.category_id = 'MUSIQUE_ENREGISTREE'  THEN "MUSIC" 
```

Comment:
```suggestion
            WHEN subcategories.category_id IN ('MUSIQUE_LIVE','MUSIQUE_ENREGISTREE') THEN "MUSIC"
```
C'est vraiment un détail